package.yaml

```diff
@@ -1,5 +1,5 @@
 name:        xss-sanitize
-version:     0.3.5.7
+version:     0.3.5.8
```

Comment:
The given version bump is a patch version bump. We'd need a minor version bump to follow the PVP (bumping the C in A.B.C.D). In other words: 0.3.6.

Reply:
Oops, sorry about that. Fixed.